baselines/single/data_mixup/data_mixup.py

```diff
@@ -0,0 +1,44 @@
+from tqdm import tqdm
+from dckit import read_datasets, random_split_data, evaluate
+import random
+import numpy as np
+
+
+def data_mix(data, num_mix=3):
+    json_data = data['json']
+    new_json = json_data
+    # 按类聚合数据
+    sentence_by_class = {}
+    label_desc_map = {}
+    for idx, tmp in enumerate(tqdm(json_data)):
+        if tmp['label'] not in label_desc_map:
+            label_desc_map[tmp['label']] = tmp['label_des']
+        if tmp['label'] not in sentence_by_class:
+            sentence_by_class[tmp['label']] = []
+        sentence_by_class[tmp['label']].append(tmp['sentence'])
+    idx = 0
+    for classes, sentences in tqdm(sentence_by_class.items()):
+        for _ in range(10000//118):
```

Comment:
Please do not use a static number like 118, you can use the length of data['info']